clients/fossid-webapp/src/main/kotlin/model/Project.kt

```diff
@@ -40,7 +41,9 @@ data class Project(
     val description: String?,
     val comment: String?,
 
-    val isArchived: Int,
+    @JsonDeserialize(using = IntBooleanDeserializer::class)
```

Comment:
commit-msg: Maybe replace in title "Change to support FossID 2024.2" with just "Support FossID 2024.2"